src/Controls/src/Core/Compatibility/Handlers/Shell/Android/SearchHandlerAppearanceTracker.cs

```diff
@@ -83,6 +84,11 @@ protected virtual void SearchHandlerPropertyChanged(object sender, System.Compon
 			}
 		}
 
+		void EditTextFocusChange(object s, AView.FocusChangeEventArgs args)
+		{
+			_searchHandler.SetIsFocused(_editText.IsFocused);
```

Comment:
Mnnn, here, should show the keyboard if `IsFocused` is true and hide the keyboard in other case.
Have extension methods to do it in the ContextExtensions class: https://github.com/dotnet/maui/blob/d91ba8e19ecae1d03f5a84a454c113582bf2c749/src/Core/src/Platform/Android/ContextExtensions.cs#L80